### PR TITLE
890 refactoring tables

### PR DESCRIPTION
**Pull request by p-netm**
Refactor Table components in the following pages:
- planning pages,
- monitor pages.

### CODE REVIEW

**Comment by p-netm** [comment]
image

**Comment by p-netm** [comment]
image

**Comment by p-netm** [comment]
image

**Comment by p-netm** [comment]
image

File: src/containers/pages/FocusInvestigation/active/index.tsx [hunk 1]
```diff
@@ -175,19 +196,122 @@ class ActiveFocusInvestigation extends React.Component<
...
+        Header: END_DATE,
+        accessor: 'plan_effective_period_end',
+      },
+    ];
+
+    const createTableProps = <D extends object>(
+      columns: Array<Column<D>>,
+      data: Plan[] | null,
+      queryParam: string
+    ) => {
```
Comment (cKellyDesign) [inline src/containers/pages/FocusInvestigation/active/index.tsx:294]: @p-netm would it make sense to move this out of the render function so we don't re-declare it every render? Also, docstrings pls 🙏

File: src/helpers/utils.tsx [hunk 2]
```diff
@@ -94,7 +94,7 @@ export interface GeoJSON {
 export function getLocationColumns(
   locations: LocationItem[] = locationHierarchy,
   padHeader: boolean = false
-): Column[] {
+) {
```
Comment (cKellyDesign) [inline src/helpers/utils.tsx:97]: If not `Column[]`, what is being returned?  Can we add params and returns to the docstring?

Comment (p-netm) [reply src/helpers/utils.tsx:97]: ok, this function is being used by 2 versions of React-table with different type declarations for Column, a union intersection will also not suffice as a solution here.

File: src/containers/pages/FocusInvestigation/active/index.tsx (hunk 1 above)
Comment (p-netm) [reply src/containers/pages/FocusInvestigation/active/index.tsx:294]: cool

File: src/helpers/utils.tsx (hunk 2 above)
Comment (cKellyDesign) [reply src/helpers/utils.tsx:97]: Ok great, that makes sense 👍

**Pull request merged**